rma/models/rma.py

```diff
@@ -732,6 +732,10 @@ def action_confirm(self):
         self = self.filtered(lambda rma: rma.state == "draft")
         if not self:
             return
+        if self.env.company.rma_reception_grouping:
```

Comment:
```suggestion
        if not self.procurement_group_id:
```

Reply:
You may set this value manually if you are interested to put a specific group, so let's keep this option open.